tests/integration/WatchdogTest.php

```diff
@@ -5,13 +5,15 @@
 /**
  * @group commands
  */
-class WatchdogCase extends CommandUnishTestCase
+class WatchdogTest extends UnishIntegrationTestCase
 {
 
     public function testWatchdog()
     {
-        $this->setUpDrupal(1, true);
-        $this->drush('pm-enable', ['dblog']);
+        $this->drush('pm-enable', ['dblog'], [], self::IGNORE_EXIT_CODE);
```

Comment:
Why ignore exit code here?

Reply:
We might enable modules again and again. I guess that enabling an already-enabled module is not an error, so we can remove this.